### ISSUE TITLE
Non-transpiled async functions fail to parse

### ISSUE THREAD

**Issue by Karolusrex** [jspm/jspm-cli#2341]
**JSPM Version:** `$ jspm --version`

0.17.0-beta.31

**Transpiler Plugin(s):** babel / traceur / typescript / I do not use a transpiler

babel

**Details:**

When turning off transformation of async functions, by modifying the babelOptions in jspm.config.js, the build fails due to parsing error in the rollup library. 

The output when trying to build is then something like this:

` err  SyntaxError: Unexpected token (354:10) in [path/to/filename.js]`

My fix: Adding to systemjs-builder/lib/builder.js on [this line](https://github.com/systemjs/builder/blob/master/lib/rollup.js#L232) and inserting the options 
```
acorn: {
            allowReserved: true,
            ecmaVersion: 8
        }
```

I'm not too deep into the code, don't really feel to sure if making a pull request on this is the right way to go. Furthermore, I might be posting this issue in the wrong repo. In that case, my apologies.

Thanks!

Karl

**Comment by guybedford** [comment]
@Karolusrex thanks so much for sharing, do you think you could provide a PR to the builder with this fix?

**Comment by Karolusrex** [comment]
Alright, sure. https://github.com/systemjs/builder/pull/825

**Comment by guybedford** [comment]
Thanks! The release cycle will likely be in the next week or so.

**Comment by Karolusrex** [comment]
Thanks a lot for your work

**Comment by guybedford** [comment]
Released in 0.17.0-beta.44.
